CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+Thank you for contributing!
+
+## Authoring
+
+Code should be well documented and tested. Each loader must:
+
+* Document for users via its own README
+* Document for developers via code comments
+* [Type-check via jsdoc](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html)
+* Cover at least 90% of code between unit and end-to-end tests
+  * unit tests for branching and especially failure cases
+  * end-to-end for [happy-paths](https://en.wikipedia.org/wiki/Happy_path)
+  * You can run tests locally via `npm run test --workspace=packages/LOADER_NAME`
+* Adhere to code styles
+  * Code can be validated with `node --run pre-commit` from the project root (CI will verify adherence but will not auto-fix).
```

Comment:
maybe mention that we use `biome` and add link if people wanna know

Reply:
I hope they can read a package.json 😅

Reply:
in nodejs.org contributing.md we put a lot of link to redirect to which tools we use. So idk maybe add footnote about our tools

Reply:
That sounds like a lotta make-work (eg when we change tools, then we also have to update the doc) for something they can already see for themselves.

Reply:
Okay let's not mention it